### PR TITLE
Lgpd webhooks testing tool

### PR DESCRIPTION
**Pull request by ferrari-lucas**
Creation of the collection with the LGPD Webhooks testing tool. The README file explains the step-by-step to use the methods arranged to send the events related to the 3 different webhooks (store/redact, customers/redact, customers/data_request)
Disclaimer: This version does not include the information about the object data_request for the customers/date_request webhook.

### CODE REVIEW

File: utils/lgpd-webhooks-testing-tool/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+Postman Collection for LGPD Webhooks Testing
+==============================================
+
+Scope
+----------------
+
+Provide a palliative mechanism for testing LGPD webhooks for partners, as the triggering scenarios for each event depend on time or action via store Admin which are not always available at design/development time.
```
Comment (Nahpal) [inline utils/lgpd-webhooks-testing-tool/README.md:7]: Let's change `palliative` for `alternative` please

**Review by Nahpal — CHANGES_REQUESTED** [review]
Awesome!

Just one tiny change.

File: utils/lgpd-webhooks-testing-tool/README.md (hunk 1 above)
Comment (ferrari-lucas) [reply utils/lgpd-webhooks-testing-tool/README.md]: Done

**Review by Nahpal — APPROVED** [review]
Thanks!

**Pull request merged**